src/test/java/KeyProviderTest.java

```diff
@@ -26,32 +30,52 @@
   private static final String SAMPLE_KEY = "sampleKey";
   private static final String CAPITALISED_SAMPLE_KEY = "SAMPLEKEY";
   private static final String INVALID_KEY = "invalidKey";
-
   private static final String SAMPLE_VALUE = "sampleValue";
+  private static final String KEYS_JSON = "{\"sampleKey\" : \"sampleValue\"}";
+  private static final File file = new File("src/main/resources/TEST_KEYS.json");
+
+  @Before
+  public void init() {
```

Comment:
Use the canonical name `setUp` for the `@Before` function - It's a standard naming convention in JUnit.

Reply:
Thanks for letting me know, I'll be sure to keep this in mind going forward!

Reply:
Thanks for letting me know, I'll be sure to keep that in mind going forward!